torchrec/distributed/tests/test_fused_embedding_bag_collection.py

```diff
@@ -38,7 +38,7 @@
     FusedEmbeddingBagCollection,
 )
 from torchrec.sparse.jagged_tensor import KeyedJaggedTensor
-from torchrec.test_utils import skip_if_asan_class
+from torchrec.test_utils import skip_if_asan_class, skipIfRocm
```

Comment:
can you run lintrunner manually? I belive this import isn't needed and lnting should catch